content/200-concepts/100-components/07-preview-features/100-sql-server/index.mdx

```diff
@@ -61,9 +67,76 @@ To make local development easier on MacOS, you can add `encrypt=DANGER_PLAINTEXT
 
 > **Note**: `encrypt=DANGER_PLAINTEXT` must not be used in production.
 
-###  Migrations are not supported yet
+###  Prisma Migrate (Preview) caveats
+
+Prisma Migrate (Preview) is supported in 2.13.0 and later with the following caveats:
+
+#### Native types
+
+Prisma Migrate (Preview) does not yet support native types, which means that you cannot use SQL Server, Prisma Migrate, and native types together.
```

Comment:
Should be can't use Prisma Migrate and Native Types together. SQL Server mention can be deleted.